lib/camp.js

```diff
@@ -629,13 +629,16 @@ function respondWithFile(req, res, path, ifNoFile) {
     }
     res.mime(p.extname(realpath).slice(1));
 
-    // Cache management.
-    if (+(new Date(req.headers['if-modified-since'])) >= +stats.mtime) {
-      res.statusCode = 304;
-      res.end();  // not modified.
+    // Cache management (compare timestamps at second-level precision).
+    var modified = new Date(stats.mtime);
```

Comment:
`stats.mtime` is already a Date.

Also, could you call it `lastModified`? It makes the if condition clearer.